client/verta/tests/registry/model_version/test_find.py

```diff
@@ -40,24 +38,18 @@ def test_find(self, client, created_entities):
         versions[name + "2"].add_label(tag_name)
         versions[name + "2"].add_label("label2")
 
-        for version in versions:
-            versions[version] = registered_model.get_version(version)
+        for version in versions.values():
+            version._refresh_cache()
```

Comment:
The `get_version()` was originally written to make sure we had up-to-date local objects, before our caching mechanisms existed. Now, `_refresh_cache()` will do the same thing.